lib/client.js

```diff
@@ -2502,6 +2568,10 @@ function checkTurnServers(client) {
     if (!client._supportsVoip) {
         return;
     }
+    if (client.isGuest()) {
+        return; // guests can't access TURN servers
```

Comment:
Nobody's got back to me so from a code PoV this is fine, and that I think is all we care about for purposes of PR review.